README.md

```diff
@@ -207,12 +207,12 @@ A. [ Disclaimer: Here, how to make the completion code visible to
    `bash-completion-config-version.cmake` files. Example usage:
 
    ```cmake
-   include(GNUInstallDirs)
    find_package(bash-completion)
    if(BASH_COMPLETION_FOUND)
      message(STATUS
        "Using bash completion dir ${BASH_COMPLETION_COMPLETIONSDIR}")
    else()
+     include(GNUInstallDirs)
```

Comment:
This is fairly cosmetic I suppose and I don't know if it actually works, but would make sense to me. It's a separate commit we can remove if it doesn't.

Reply:
I don't use cmake, so I'm not familiar with it. AI says it works, and AI is probably right. However, I'm not sure whether there is anything that we should be careful with. I think the original author @jtojnar should know this better.

Reply:
I believe conditional imports are fine in CMake should probably work. Though, in practice, it [should be used elsewhere](https://github.com/jtojnar/cmake-snips#hardcoding-the-installation-paths) in the CMake script too.